pandas/api/test.py

```diff
@@ -0,0 +1,23 @@
+"""
+Entrypoint for testing from the top-level namespace
+"""
```

Comment:
I would keep this in `util` (but maybe `util/_tester.py` instead of `util/nosetester.py`

In any case, IMO this is a strange place, as we currently use `pd.api` to put things from other places in a consistent place, not to put actual code for which the entry point for users is not here.